doc/assets/css/common/global.scss

```
@@ -157,3 +157,13 @@ table.fixed td { overflow: auto; }
   margin-top: -6rem;
   }
 }
+
+.header-hash {
+  font-size: 100%;
+  visibility: hidden;
+  color:silver;
+}
+
+h2:hover a, h3:hover a, h4:hover a, h5:hover a, h6:hover a {
```

```suggestion
h2, h3, h4, h5, h6 {
  &:hover a.header-hash {
    visibility: visible
  }
}
```

Also, be mindful about applying styles globally to entire tags. I guess it's fine here but usually should be avoided.